docs/sdk/src/reference_docs/transaction_extensions.rs

```diff
@@ -0,0 +1,57 @@
+//! Transaction extensions are, briefly, a means for different chains to extend the "basic"
```

Comment:
nit: try using `git mv`, or a combination of `git rm` and `add` to avoid such renames not being picked up by git.